CONFIGURATION.md

```
@@ -0,0 +1,589 @@
+
+```bash
+git add environments/
+git commit -m "Add production environment config"
+git push
+```
+
+### Use Meaningful Filenames
+
+Name files descriptively:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)